Fabric.Terminology.API/Scripts/Terminology-Intall-Utilities.psm1

```
@@ -0,0 +1,135 @@
+}
+
+function Get-TerminologyConfig{
+    param(
+        [string] $Path
+    )
+    
+    $installSettings = Get-InstallationSettings "terminology"
+
+    $iisUser = Read-Host "Press Enter to accept the default IIS App Pool User"
+    $userEnteredPassword = Read-Host "Enter the password for $iisUser" -AsSecureString
+    $credential = New-Object -TypeName "System.Management.Automation.PSCredential" -ArgumentList $iisUser, $userEnteredPassword
+    [System.Reflection.Assembly]::LoadWithPartialName("System.DirectoryServices.AccountManagement") | Out-Null
+    $ct = [System.DirectoryServices.AccountManagement.ContextType]::Domain
+    $pc = New-Object System.DirectoryServices.AccountManagement.PrincipalContext -ArgumentList $ct, $credential.GetNetworkCredential().Domain
+    $isValid = $pc.ValidateCredentials($credential.GetNetworkCredential().UserName, $credential.GetNetworkCredential().Password)
+    if (!$isValid) {
+        Write-Error "Incorrect credentials for $iisUser"
+        throw
+    }
+
+    $config = [TerminologyConfig]::new()
+    $config.iisUserCredentials = New-Object System.Management.Automation.PSCredential ($iisUser, $userEnteredPassword)
+    $config.appName = $installSettings.appName
+    $config.appPool = $installSettings.appPool
+    $config.siteName = $installSettings.siteName
+    if ([string]::IsNullOrWhiteSpace($installSettings.discoveryServiceUrl)) {
+        $config.discoveryServiceUrl = Read-Host "Please enter the discovery service Uri (eg. https://SERVER/discoveryservice/v1)"
+    } else {
+        $config.discoveryServiceUrl = $installSettings.discoveryServiceUrl
+    }
+    
+
+	return $config
+}
+
+function Update-AppSettings{
+    param(
+        [TerminologyConfig] $config
+    )
+
+    $appPath = Get-WebURL "IIS:\Sites\$($config.siteName)\$($config.appName)"
+    $config.applicationEndpoint = $appPath.ResponseUri
+
+    $appSettings = "$(Get-IISWebSitePath -WebSiteName $config.siteName)\$($config.appName)\appsettings.json"
```

This requires the WebAdminModule to be imported - maybe consider attempting the import at the top of the module so you can surface the error in a easier to digest fashion?